tests/functional/cli_test.py

```diff
@@ -9,3 +18,168 @@ def test_help():
 def test_version():
     """Test the slack-annotations --version command."""
     run(["slack-annotations", "--version"], check=True)
+
+
+@pytest.fixture
+def slack_annotations():
+    return {
+        "text": "2 new annotations",
+        "blocks": [
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": "`test_user_1` (md............................) annotated <https://example.com/|Annotating the law | Hypothes.is>:",
+                },
+                "fields": [
+                    {"type": "mrkdwn", "text": "*Quote:*"},
+                    {
+                        "type": "mrkdwn",
+                        "text": "*Annotation* (<https://hyp.is/test_annotation_id_1/example.com/|in-context link>):",
+                    },
+                    {"type": "plain_text", "text": "(None)"},
+                    {"type": "plain_text", "text": "test_user_1 reply"},
+                ],
+            },
+            {"type": "divider"},
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": "`test_user_2` (Test User 2) annotated <https://web.hypothes.is/blog/step-by-step-guide-to-using-hypothesis-for-collaborative-projects/|Annotating the law | Hypothes.is>:",
+                },
+                "fields": [
+                    {"type": "mrkdwn", "text": "*Quote:*"},
+                    {
+                        "type": "mrkdwn",
+                        "text": "*Annotation* (<https://hyp.is/test_annotation_id_2/web.hypothes.is/blog/step-by-step-guide-to-using-hypothesis-for-collaborative-projects/|in-context link>):",
+                    },
+                    {
+                        "type": "plain_text",
+                        "text": "The ability to collaborate effectively is invaluable in today\u2019s interconnected world. Whether in academic or professional settings, tools that streamline communication, enhance content sharing and promote open dialogue are important. Hypothesis is a standout tool in this regard, especially renowned for its capabilities in group annotations and as an online learning tool.",
+                    },
+                    {"type": "plain_text", "text": "A useful tool!"},
+                ],
+            },
+            {"type": "divider"},
+            {
+                "type": "context",
+                "elements": [
+                    {
+                        "type": "mrkdwn",
+                        "text": "These annotations are posted to Slack by a <https://github.com/hypothesis/slack-annotations/|GitHub Actions workflow>",
+                    }
+                ],
+            },
+        ],
+    }
+
+
+@pytest.fixture
+def search_annotations():
+    return {
+        "rows": [
+            {
+                "id": "test_annotation_id_1",
```

Comment:
We can get rid of `"id"` and `"updated"` from both annotations, they're not used.